src/main/java/teammates/common/util/FieldValidator.java

```diff
@@ -8,6 +8,7 @@
 import org.joda.time.DateTimeZone;
 
 import teammates.common.datatransfer.FeedbackParticipantType;
+import teammates.ui.controller.PageData;
```

Comment:
This dependency should not occur. UI package should not be accessible by any other package, especially the common package.
You can move all the methods needed to `FieldValidator` itself, or create a new class e.g `NationalityHelper`, up to your preference, but do not create this kind of dependency.